### ISSUE TITLE
Multiple resource folders in balloon

### ISSUE THREAD

**Issue by Ran-n** [kaiiiz/hugo-theme-monochrome#13]
Hi, i was wondering if there is or would you consider adding the possibility to list multiple folders in the ballon_resources option.

**Comment by kaiiiz** [comment]
Yes, this feature can be implemented. However, I'm curious about what is the use case of having multiple resource folders. Can you show me some examples?
